fragments/testcontainers-postgresql/accelerator.yaml

```diff
@@ -0,0 +1,72 @@
+engine:
+  merge:
+
+    - include: [ "pom.xml" ]
+      chain:
+        - type: ReplaceText
+          condition: "#databaseIntegrationTestType == 'testcontainers'"
```

Comment:
The `databaseIntegrationTestType` option isn't defined in this fragment (or imported anywhere) so it shouldn't be used. 
Using an option defined in the parent accelerator sometimes works, but is inconsistent and isn't formally supported.

Reply:
hmm, I thought I removed all of those references but I see now there where some stuff left behind after getting moved to another fragment - will fix